### PR TITLE
Change Passport to Fetch Own Application

### PR DESCRIPTION
**Pull request by hunterchen7**
closes nothing

# Quick Overview of Changes

- change passport component to fetch it's own application

# Checklist

- [x] If any frontend changes were made, include a screenshot/demo in the overview
- [x] Checked all uses of changed component(s)/function(s)
- [x] If any changes were made to our backend services, at least one (happy path) unit test was added OR a `TODO` comment was added to create one.
- [x] Check that CI is passing.
- [x] If no (re-)reviews after 1-2 days, ping the web leads on the discord to remind them to review the PR.

@basokant @ArsalaanAli

### CODE REVIEW

File: src/components/apply/avatar.tsx [hunk 1]
```diff
@@ -30,18 +30,17 @@ function getAvatarImage(avatar: AvatarType) {
...
-      />
-      <div className="absolute left-[2%] top-[47%] -rotate-[8.646deg] transform items-center justify-center gap-[10px] border border-[#DCDFE9] bg-[#F8F5FF] p-[4px_8px]">
-        <p className="font-sans text-[13px] font-medium leading-[18px] text-[#976CDF]">
-          {avatar}
-        </p>
-      </div>
+    <div className="relative flex aspect-square w-2/5 flex-col items-center justify-end rounded-lg bg-primary-200 pr-[18.896px]">
+      {avatar && (
+        <>
+          <div className="absolute -left-3 top-0 z-50 -rotate-[8.646deg] transform items-center justify-center gap-[10px] border border-[#DCDFE9] bg-[#F8F5FF] p-[4px_8px]">
```
Comment (basokant) [inline src/components/apply/avatar.tsx:36]: heyo, do this or i'll kill ur first born son

File: src/components/apply/avatar.tsx [hunk 2]
```diff
@@ -30,18 +30,17 @@ function getAvatarImage(avatar: AvatarType) {
...
-      />
-      <div className="absolute left-[2%] top-[47%] -rotate-[8.646deg] transform items-center justify-center gap-[10px] border border-[#DCDFE9] bg-[#F8F5FF] p-[4px_8px]">
-        <p className="font-sans text-[13px] font-medium leading-[18px] text-[#976CDF]">
-          {avatar}
-        </p>
-      </div>
+    <div className="relative flex aspect-square w-2/5 flex-col items-center justify-end rounded-lg bg-primary-200 pr-[18.896px]">
+      {avatar && (
+        <>
+          <div className="absolute -left-3 top-0 z-50 -rotate-[8.646deg] transform items-center justify-center gap-[10px] border border-slate-200 bg-primary-100 p-[4px_8px]">
```
Comment (basokant) [inline src/components/apply/avatar.tsx:36]: can you change the p-[4px_8px[ to p-4 p-8

Comment (hunterchen7) [reply src/components/apply/avatar.tsx]: no

File: src/components/apply/passport-field.tsx [hunk 3]
```diff
@@ -1,15 +1,16 @@
 import React from "react";
 
 type FieldType = {
-  str?: string | null;
-  int?: number | null;
+  value?: string | null;
```
Comment (basokant) [inline src/components/apply/passport-field.tsx:4]: can u remove ?

**Review by basokant — CHANGES_REQUESTED** [review]
image

File: src/components/apply/passport-field.tsx (hunk 3 above)
Comment (basokant) [reply src/components/apply/passport-field.tsx:4]: add | undefined, ? is poo poo

**Review by basokant — APPROVED** [review]

File: src/components/apply/passport-field.tsx (hunk 3 above)
Comment (hunterchen7) [reply src/components/apply/passport-field.tsx]: no

**Pull request merged**